src/secp256k1.c

```diff
@@ -249,6 +249,8 @@ static int secp256k1_pubkey_load(const secp256k1_context* ctx, secp256k1_ge* ge,
         secp256k1_fe x, y;
         secp256k1_fe_set_b32_mod(&x, pubkey->data);
         secp256k1_fe_set_b32_mod(&y, pubkey->data + 32);
+        secp256k1_fe_normalize_var(&x);
+        secp256k1_fe_normalize_var(&y);
```

Comment:
Now that I see what `secp256k1_pubkey_save` does, we should simply do this: 
```suggestion
        ARG_CHECK(secp256k1_fe_set_b32_limit(&x, pubkey->data));
        ARG_CHECK(secp256k1_fe_set_b32_limit(&y, pubkey->data + 32));
```

Here, `ARG_CHECK(cond)` is a macro that calls the "illegal-argument callback" if `cond` is false, see https://github.com/bitcoin-core/secp256k1/blob/67214f5f7d8e0bdf193ceb1f47ba8277d1a0871a/include/secp256k1.h#L317
It's also used below for the `!secp256k1_fe_is_zero(&ge->x)` check that triggers the test failure.